app/scripts/directives/deleteLink.js

```diff
@@ -165,9 +173,9 @@ angular.module("openshiftConsole")
               if (!promises.length) {
                 navigateToList();
               } else {
-                // Wait until all promises resolve so that we can add alerts to
-                // AlertMessageService before navigating, otherwise they aren't
-                // displayed.
+                // Wait until all promises resolve so that we can add
+                // notifications to NotificationsService before navigating,
```

Comment:
is this comment still true? the notification service lives outside the view.  Still seems reasonable that you would wait to navigate until the delete is actually successful, i think the comment just needs updating.